### DISCUSSION TITLE
Unable to find a compatible cbor library.

### DISCUSSION THREAD

**Discussion by Teja25-web** [fastlane/fastlane#22194] [Q&A]
I am using the following version for fastlane
ruby - 3.3.4 (also tried 3.0.4)
fastlane - 2.221.1 (also tried 2.222.0)

I am building an image in docker container. My base image is of RHEL8. 
Fastlane and ruby installations are successful. All gems are installed successfully.

But when I run fastlane init command from the projects root directory I am getting the following error:
/usr/local/bundle/gems/aws-sdk-core-3.201.5/lib/aws-sdk-core/cbor.rb:85:in `set_default_engine': Unable to find a compatible cbor library. (RuntimeError)

I have tried installing CBOR library but no luck. 
This setting was working few months back but not any more.
Also, when I try to use the same versions and setup fastlane on my mac, it works fine.

Any help would be appreciated.

**Comment by seba9999** [comment]
FYI this [SO comment](https://stackoverflow.com/a/75094719) solved the same problem for me ;) 

Simply add rexml to your gemfile
